src/main/scala/abalone/format/Forsyth.scala

```diff
@@ -59,15 +59,18 @@ object Forsyth {
     }
 
   def >>(game: Game): FEN = {
-    val boardFen   = getFen_board(game.situation.board)
-    val scoreStr   = game.situation.board.history.score.fenStr
-    val player     = game.situation.player.fold('b', 'w')
-    val prevPlayer = if (game.situation.board.variant.hasPrevPlayer) {
-      game.situation.board.history.prevPlayer.fold(" *")(p => p.fold(" b", " w"))
+    val boardFen  = getFen_board(game.situation.board)
+    val scoreStr  = game.situation.board.history.score.fenStr
+    val player    = game.situation.player.fold('b', 'w')
+    val halfMoves = game.situation.board.history.halfMoveClock
+    val fullMoves = game.fullTurnCount
+    val prev      = if (game.situation.board.variant.hasPrevPlayer) {
+      val lastTurn    = game.situation.board.history.lastTurn.reverse.headOption
+      val currentTurn = game.situation.board.history.currentTurn.reverse.headOption
+
+      " " + currentTurn.fold("-")(_.uci) + " " + lastTurn.fold("-")(_.uci)
```

Comment:
To keep consistency, we just want currentTurn to be put in after the `½` character. 
We also don't need a '-' if no currentTurn.
Something like
```
val currentTurn = game.situation.board.history.currentTurn.map(_.uci).mkString(",")
if (currentTurn.size > 0) s" ½$currentTurn" else ""
```